packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/controller.js

```diff
@@ -0,0 +1,9 @@
+export default class PciNewProjectPromotionBannerController {
+  isPromotionBannerDisplayed() {
+    return this.model && this.model.voucher && this.model.voucher.valid;
+  }
+
+  getVoucherCredit() {
+    return this.model && this.model.voucher && this.model.voucher.credit.text;
```

Comment:
Same as https://github.com/ovh/manager/pull/8036/files#r979409773
```suggestion
    return this.model?.voucher?.credit?.text;
```

Reply:
Done.